staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -47,7 +47,12 @@ EXT_APIS_PKG="$4"
 GROUPS_WITH_VERSIONS="$5"
 shift 5
 
-go install ./"$(dirname "${0}")"/cmd/{defaulter-gen,conversion-gen,client-gen,lister-gen,informer-gen,deepcopy-gen,openapi-gen}
+(
+  # To support running this script from anywhere, we have to first cd into this directory
+  # so we can install the tools.
+  cd "$(dirname "${0}")"
```

Comment:
Better use simple shell built-in parameter expansion
```shell
cd "${0%/*}"
```
rather than calling `dirname`.